app/api/routes/product_routes.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+from typing import List
+from uuid import UUID
+from fastapi import APIRouter, HTTPException, Depends
+
+from app.models import User 
+from app.product_model import Product, CreateProductRequest, UpdateProductRequest
+from app.api.auth import get_current_admin
+
+router = APIRouter()
+
+products_db = {
+    UUID("1e2d4567-e89b-12d3-a456-426614174000"): {
+        "id": UUID("1e2d4567-e89b-12d3-a456-426614174000"),
+        "name": "Product A",
+        "description": "Description of Product A",
+        "price": 100.0,
+        "created_at": datetime.now(),
+        "updated_at": datetime.now(),
+    },
+    UUID("2f3d4567-e89b-12d3-a456-426614174001"): {
+        "id": UUID("2f3d4567-e89b-12d3-a456-426614174001"),
+        "name": "Product B",
+        "description": "Description of Product B",
+        "price": 150.0,
+        "created_at": datetime.now(),
+        "updated_at": datetime.now(),
+    },
+}
+
+# Create product (only admin)
+@router.post("/products", response_model=Product)
+async def create_product(product: CreateProductRequest, current_admin: User = Depends(get_current_admin)):
+    new_product = Product(**product.dict(), created_at=datetime.now(), updated_at=datetime.now())
+    products_db[new_product.id] = new_product.dict()
```

Comment:
We need to verify if the product already exists. Please take a look at the ticket for all the details.